lib/amqp/application/channel.ex

```diff
@@ -151,6 +154,10 @@ defmodule AMQP.Application.Channel do
     {:stop, reason, %{state | channel: nil, monitor_ref: nil}}
   end
 
+  def handle_info({ref, _res}, state) when is_reference(ref) do
+    {:noreply, state}
+  end
```

Comment:
Can you add the same handler to `AMQP.Application.Connection`?

Reply:
Added